lib/routes.json

```diff
@@ -3917,12 +4025,6 @@
         "org": {
           "required": true,
           "type": "string"
-        },
-        "page": {
-          "type": "integer"
-        },
-        "per_page": {
-          "type": "integer"
```

Comment:
this is incorrect but will be addressed by the GitHub docs team in future. Not very harmful to have that removed for now, a temporary workaround is not worth it.